tests/tests_unit.yml

```diff
@@ -29,18 +43,31 @@
       ignore_errors: true
       register: python2_available
 
-    - command: python2 /tmp/test-unit-1/test_network_connections.py
+    - command: python2 /tmp/test-unit-1/test_network_connections.py --verbose
       when: python2_available | succeeded
-
+      register: python2_result
 
     - package:
         name: python3
         state: present
       ignore_errors: true
       register: python3_available
 
-    - command: python3 /tmp/test-unit-1/test_network_connections.py
+    - command: python3 /tmp/test-unit-1/test_network_connections.py --verbose
       when: python3_available | succeeded
+      register: python3_result
+
+    - command: python /tmp/test-unit-1/test_network_connections.py --verbose
+      register: python_result
+
```

Comment:
Is there a need to invoke bare `python` when there are `python2` and `python3` invocations?

Reply:
It is the only command that is guaranteed to be called, if there is no python2 or python3 (e.g. no symlink for these), then no tests would be called.

Reply:
Can it really happen (on systems that we support) that `python` exists and neither `python2` nor `python3` do?
By the way, I would not call `python` "guaranteed" in the future, given https://fedoraproject.org/wiki/Changes/Move_usr_bin_python_into_separate_package

Reply:
Hm, I guess I will address this eventually, we can probably add something that fails explicitly if not at least one binary was called. I would like to address this in the future, maybe I will change the way the unit tests are called anyhow.